needle-slack-connector/src/app/_components/DeleteConnectorButton.tsx

```diff
@@ -0,0 +1,27 @@
+"use client";
+
+import { useRouter } from "next/navigation";
+import { api } from "~/trpc/react";
+
+export function DeleteConnectorButton({
+  connectorId,
+}: {
+  connectorId: string;
+}) {
+  const router = useRouter();
+  const { mutate: deleteZendeskConnector } = api.connectors.delete.useMutation({
```

Comment:
refactoring artifact :) let's rename all zendesk-ish stuff to slack

Reply:
Oops 😅